app/views/users/show.html.erb

```diff
@@ -6,4 +6,30 @@
       <%= render @posts %>
     </ul>
   </article>
+
+  <ol>
+  <% @user.friends.each do |friend| %>
+    <li>
+      <%= friend.name %>
+      <% if @user == current_user %>
+        <%= link_to 'Unfriend',
+        friendships_destroy_path(friendship_id: Friendship.find_friendship(@user.id, friend.id))  %>
+      <% end %>
+    </li>
```

Comment:
Remove logic from your views. Please, move them in view helpers.

Reply:
completed the 1, 4, 5 reviews made by tse. 2 & 3 are not part of the microverse requirments.

Reply:
completed the 1, 4, 5 reviews made by tse. 2 & 3 are not part of the microverse requirements.